test/unit/sql/index.test.js

```diff
@@ -301,12 +301,10 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
   describe('removeIndex', () => {
```

Comment:
You can remove this test here. If it's not tested in `remove-index-query.test.ts` you can move it there